src/theia/sfm/reconstruction_builder.h

```diff
@@ -146,6 +146,9 @@ class ReconstructionBuilder {
                        const std::string& image2,
                        const ImagePairMatch& matches);
 
+  // Assign a mask for each image.
+  bool SetMasksForFeaturesExtraction(const std::vector<std::string>& mask_filepaths);
```

Comment:
Hmmm for the cpp line usually I have the entire -> on the first line then the rest on the next line. I think it would help readability if you indented the second line 2 spaces though.